queue_services/entity-digital-credentials/src/entity_digital_credentials/digital_credentials_processors/change_of_registration.py

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Processing change of registration actions."""
+
+from legal_api.models import Business, DCDefinition, DCRevocationReason, Filing
+
+from entity_digital_credentials.helpers import get_issued_digital_credentials, replace_issued_digital_credential
+
+
+async def process(business: Business, filing: Filing):
+    """Process change of registration actions."""
+    if filing.filing_json.get('filing').get(filing.filing_type).get('nameRequest') is not None:
+
+        issued_credentials = get_issued_digital_credentials(business=business)
+        if not (issued_credentials and len(issued_credentials)):
+            raise Exception('No issued credentials found.')
```

Comment:
All the processors essentially have this logic of raising an exception when no issued credentials are found.  

This will mean we're going to have a lot of error messages in the queue logs for filings where the business does not have a digital credential.  

Should this just be a log message and we just return when no digital credential is found?

Reply:
I've looked at the latest code and it logs an error now.  

For a business that legitimately doesn't use a digital credential, I don't think it actually makes sense to log this as an error message.

Maybe a warning statement? 

What do you think?

Reply:
Yeah you're right, we'll log it as a warning instead